planetcantile/data/generate.py

```diff
@@ -132,15 +132,14 @@ def __post_init__(self):
         pass
         #print(f'Could not find authority for {crs_obj.to_wkt()}')
 
-
-for tmsp in crss:
+for tmsp in crss[10:]:
     # create the tms object
     tms = morecantile.TileMatrixSet.custom(**asdict(tmsp))
-    tmsj = tms.json(exclude_none=True)
-    tmsj = tmsj.replace(CRS_to_uri(tms.supportedCRS), CRS_to_urn(tms.supportedCRS), 1)
-    tmsj = tmsj.replace(CRS_to_uri(tms.boundingBox.crs), CRS_to_urn(tms.boundingBox.crs), 1)
-    _d = json.loads(tmsj)  # get to pretty printed json
+    tmsj = tms.dict()
+    # Include URN to the planetary projections; _geographic_crs is needed by downstream libs, e.g., morecantile 
+    tmsj['supportedCRS'] = tmsj['boundingBox']['crs'] = CRS_to_urn(tms.supportedCRS)
```

Comment:
I may be missing something about dict updates, but this should probably through an exception of some kind due to the extra equals sign, or maybe it sets it to null? should be possible to remove by operating on the object directly as suggested above

Reply:
I thought (?) that this is pretty standard notation when setting two variables to the same value.

```python
>>> x = y = 2
>>> print(x)
2
>>> print(y)
2
```
Also, through the object was a no go.

Reply:
huh interesting, I guess I never needed to set variables to the same value before like this.